src/pages/psa/index.md

```diff
@@ -3,6 +3,9 @@ templateKey: info-page
 path: /psa
 title: Celebrity PSAs
 ---
+
 ## Emily Hampshire
 
-<iframe width="840" height="473" src="https://www.youtube.com/embed/RsqId8YtMhQ" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
\ No newline at end of file
+`youtube:https://www.youtube.com/embed/RsqId8YtMhQ`
```

Comment:
This is great! Can you pick which one you think will work best? I'm partial to the embed-youtube.

Reply:
I'd prefer embed-youtube as well

Reply:
cool - once that's one I'll approve and merge! You are on a role!